net/shadowsocks-libev/files/ss-rules

```diff
@@ -120,6 +120,9 @@ tp_rule() {
 	$ipt_m -A SS_SPEC_TPROXY -p udp -m set ! --match-set ss_spec_wan_ac dst \
 		-j TPROXY --on-port $LOCAL_PORT --tproxy-mark 0x01/0x01
 	$ipt_m -A PREROUTING -p udp $EXT_ARGS \
+		-m socket \
+		-m comment --comment "_SS_SPEC_RULE_" -j MARK --set-mark 1
+	$ipt_m -A PREROUTING -p udp $EXT_ARGS \
```

Comment:
And if we dont mark it first it wont route those packet locally document on here suggested it too:
https://people.netfilter.org/hidden/nfws/nfws-2008-tproxy_slides.pdf